README.md

```diff
@@ -27,6 +27,8 @@ To build Spark and its example programs, run:
 (You do not need to do this if you downloaded a pre-built package.)
 More detailed documentation is available from the project site, at
 ["Building Spark"](http://spark.apache.org/docs/latest/building-spark.html).
+For developing Spark using an IDE, see [Eclipse](https://cwiki.apache.org/confluence/display/SPARK/Useful+Developer+Tools#UsefulDeveloperTools-Eclipse)
```

Comment:
The README is supposed to be short and contain pointers to more documentation on the wiki; no I don't think it's worth duplicating much here in a markdown file. It's pretty minor but if we add new IDE help in the wiki it has to be updated here too.